app/src/lib/git/stash.ts

```diff
@@ -113,10 +111,21 @@ export function createDesktopStashMessage(branchName: string) {
  */
 export async function createDesktopStashEntry(
   repository: Repository,
-  branchName: string
+  branchName: string,
+  untrackedFilesToStage: ReadonlyArray<string>
```

Comment:
i'd prefer this be `ReadonlyArray<WorkingDirectoryFileChange>` so that its harder to pass a malformed path to this function. it looks like all the callers of this are using a helper to get those paths from a list of `WorkingDirectoryFileChange`s already. we already have a `Repository` argument that is really just used for the path so this would be pretty parallel to that